documentation/modules/overview/con-configuration-points-bridge.adoc

```diff
@@ -11,6 +11,18 @@ Kafka Bridge consumer and producer configuration is standard, as described in th
 
 HTTP-related configuration options set the port connection which the server listens on.
 
+[discrete]
+== CORS
+
+The Kafka Bridge supports the use of Cross-Origin Resource Sharing (CORS).
+CORS is a HTTP mechanism that allows browser access to selected resources from more than one origin.
+You can define a list of allowed resource origins and HTTP methods for interaction with the Kafka cluster.
```

Comment:
Explain where you can define this list.

Suggested change:

for interaction with the Kafka cluster > for interaction with the Kafka cluster through the Kafka Bridge

Reply:
Updated:

> If you choose to use CORS, you can define a list of allowed resource origins and HTTP methods for interaction with the Kafka cluster through the Kafka Bridge.
> The lists are defined in the `http` specification of the Kafka Bridge configuration.